diff -up st-0.8.4/config.def.h st-0.8.4-scientificafont/config.def.h
--- st-0.8.4/config.def.h	2021-02-20 16:45:35.339791172 +0100
+++ st-0.8.4-scientificafont/config.def.h	2021-02-20 16:48:21.376455665 +0100
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "scientifica:pixelsize=12";
 static int borderpx = 2;

 /*
@@ -26,7 +26,7 @@ char *stty_args = "stty raw pass8 nl -ec
 char *vtiden = "\033[?6c";

 /* Kerning / character bounding-box multipliers */
-static float cwscale = 1.0;
+static float cwscale = 0.85;
 static float chscale = 1.0;

 /*
